index.js

```diff
@@ -126,13 +126,9 @@ cm.Item({
 });
 
 function checkLocation(opts) {
```

Comment:
Let's leave this pr as it is for now, the changes I was working on yesterday will likely effect the vine urls code now. HTML5 Video support is encapsulated in #6 but I'm fine with have a more direct issue for it